app/controllers/admin/manage_users_controller.rb

```diff
@@ -7,12 +7,9 @@ def text_namespace
     end
 
     before_action :require_user_manager!
-    before_action :set_return_url, only: [:edit, :update]
 
     def index
-      page = (index_params[:page].presence || 1)
-      per_page = Rails.configuration.x.admin.pagination_per_page
-      @users = User.all.order(first_name: :asc, last_name: :asc).page(page).per(per_page)
+      @users = User.all.order(first_name: :asc, last_name: :asc).page(params[:page])
```

Comment:
Could we make this multi-line - just a bit easier to parse :)

Reply:
Done :)